common/dbconnector.cpp

```diff
@@ -209,7 +209,7 @@ void SonicDBConfig::validateNamespace(const string &netns)
     }
 }
 
-SonicDBInfo& SonicDBConfig::getDbInfo(const std::string &dbName, const std::string &netns)
+std::unordered_map<std::string, SonicDBInfo> SonicDBConfig::getDbEntry(const std::string &netns)
```

Comment:
```suggestion
const std::unordered_map<std::string, SonicDBInfo>& SonicDBConfig::getDbEntry(const std::string &netns)
```

We need to avoid copying the map.